src/test/groovy/hudson/plugins/gradle/injection/BuildScanInjectionGradleIntegrationTest.groovy

```diff
@@ -652,23 +652,23 @@ class BuildScanInjectionGradleIntegrationTest extends BaseGradleIntegrationTest
 
         when:
         withInjectionConfig {
-            injectionVcsRepositoryPatterns = pattern
+            vcsRepositoryFilter = filter
         }
         enableBuildInjection(slave, gradleVersion)
         def build2 = j.buildAndAssertSuccess(p)
 
         then:
-        if (pattern == "simple-") {
+        if (filter.contains("simple-")) {
             j.assertLogContains(MSG_INIT_SCRIPT_APPLIED, build2)
         } else {
             j.assertLogNotContains(MSG_INIT_SCRIPT_APPLIED, build2)
         }
 
         where:
-        pattern << ["simple-", "this-one-does-not-match"]
+        filter << ["+:simple-", "+:this-one-does-not-match"]
```

Comment:
💄 Maybe use a table here, so that there is no data-specific logic in the test code?

```
filter                      | shouldApplyAutoInjection
"+:simple-"                 | true
"+:this-one-does-not-match" | false
```

Reply:
Agree, will change

Reply:
Done